src/app/components/quiz-page/quiz-page.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="content">
+  <div class="question-container">
+    <h1>{{quizModel.question.text}}</h1>
+  </div>
+  <mat-radio-group [(ngModel)]="selectedAnswer" class="form-container">
+    <mat-radio-button  class="answer-container" *ngFor="let item of quizModel.question.answers" [value]="item" (click)="select(item)">
```

Comment:
zawsze ładniej jak iterowany obiekt nazywa się tym czym jest a nie `item`. Tutaj bedzie to `answer` po prostu